WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -209,6 +211,12 @@ private async Task StepTransactionSigningPhaseAsync(CancellationToken cancellati
 						}
 						round.LogInfo($"There are {indistinguishableOutputs.Count(x => x.count == 1)} occurrences of unique outputs.");
 
+						// Store transaction.
+						if (TransactionArchiver is not null)
+						{
+							await TransactionArchiver.StoreJsonAsync(coinjoin).ConfigureAwait(false);
+						}
```

Comment:
What if there is a problem when brodcasting the transaction?
Shouldn't we store the transaction only after it was successfully broadcast?

Reply:
Since we don't do anything automatically with this (yet?) I think this behavior is preferred, if there is an ephemeral problem with broadcasting then we can manually intervene and still broadcast the successfully completed round's txn